index.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
     indent: ['error', 2, {SwitchCase: 0, MemberExpression: 0}],
     quotes: ['error', 'single', {avoidEscape: true}],
     semi: ['error', 'never'],
+    'unicorn/prevent-abbreviations': 'off',
```

Comment:
hehe, feel free to remove this i can just start typing `context` lol

Reply:
Haha it was actually messing up a bunch of stuff in `oclif/oclif`, like trying to correct `sourceDir` to `sourceDirection`.

Reply:
I just noticed! it was changing type names!